app/views/proposals/show.html.haml

```diff
@@ -25,21 +26,24 @@
 
   .row
     .col-md-3
-      .speakerinfo
-        .col-md-12
-          = image_tag @speaker.gravatar_url(:size => 200), :class => "img-responsive img-rounded"
-        .col-md-12
-          %h3
-            by
-            = link_to @speaker.name, user_path(@speaker.id)
-            = "(#{@speaker.email})" if @speaker.email_public
-            - if @speaker.affiliation?
-              %br
-              %span.muted
-                from
-                = @speaker.affiliation
-          -if @speaker.biography?
-            = markdown(@speaker.biography)
+      %h3
+        Presented by:
+      - speakers_ordered.each do |speaker|
+        .speakerinfo
+          .row
+            .col-md-4
+              = image_tag speaker.gravatar_url(:size => 120), :class => "img-responsive img-rounded"
```

Comment:
I agree we should refactor our views and while this would not be for me no 1 priority in such feature, we do try to use the new syntax as well as rubocop styling (eg. in this case single quotes) in any new code we submit.